### PR TITLE
clean: Remove all core configurations and README sections CY-2485

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

**Comment by lolgab** [comment]
There are still some dynamoDB things in the repository.
Not sure they are part of core, tho.

**Review by franciscodua — APPROVED** [review]

**Review by pauloribeiro-codacy — DISMISSED** [review]
The only change that affects the documentation is the removal of the block in the `values-production.yaml` file. It will be published on the next release of the chart. :+1:

**Comment by rtfpessoa** [comment]
DynamoDB should only be used by core, and you should also update the requirements.lock file probably. The rest looks fine.

**Comment by lolgab** [comment]
@rtfpessoa I removed all the DynamoDB parts.
About the `requirements.lock`, it is no more committed in the repository since https://github.com/codacy/chart/pull/393
Should I change the minimum codacy-website version to the first one not using core, right? Same thing should apply for other components that were using core. Correct?

File: docs/infrastructure/EKS/backend/variables.tf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # variables.tf - user settings. For provider settings see config.tf
 
 variable "project_tag" {
-  description = "Project tag to add to s3 and dynamodb names"
```
Comment (h314to) [inline docs/infrastructure/EKS/backend/variables.tf:4]: please don't delete. See comment above

File: docs/infrastructure/EKS/backend/state_and_lock.tf [hunk 2]
```diff
@@ -22,26 +22,6 @@ resource "aws_s3_bucket" "state" {
   tags = var.custom_tags
 }
 
-resource "aws_dynamodb_table" "lock" {
```
Comment (h314to) [inline docs/infrastructure/EKS/backend/state_and_lock.tf:25]: Please keep the lock table

File: docs/infrastructure/EKS/backend/outputs.tf [hunk 3]
```diff
@@ -1,7 +1,3 @@
 output "state_bucket_name" {
   value = aws_s3_bucket.state.id
 }
-
```
Comment (h314to) [inline docs/infrastructure/EKS/backend/outputs.tf:4]: Please don't remove

File: docs/infrastructure/EKS/aws-terraform-minimum-admin-policy.json [hunk 4]
```diff
@@ -62,11 +62,6 @@
       ],
       "Resource": "*"
     },
-    {
```
Comment (h314to) [inline docs/infrastructure/EKS/aws-terraform-minimum-admin-policy.json:65]: This shouldn't be removed

File: docs/infrastructure/EKS/main/config.tf [hunk 5]
```diff
@@ -12,7 +12,6 @@ terraform {
   #backend "s3" {
   #  encrypt = true
   #  bucket = "YOUR_S3_BUCKET_NAME_HERE"
-  #  dynamodb_table = "codacy-terraform-lock"
```
Comment (h314to) [inline docs/infrastructure/EKS/main/config.tf:15]: please don't remove this

**Review by h314to — CHANGES_REQUESTED** [review]

**Comment by h314to** [comment]
> DynamoDB should only be used by core, and you should also update the requirements.lock file probably. The rest looks fine.

DynamoDB is used by Terraform for state locking. A table is created specifically for that purpose. See here for further details: https://www.terraform.io/docs/backends/types/s3.html

**Review by pedrocodacy — APPROVED** [review]
pending on a review from @h314to to confirm that we are not messing with dynamo anymore

**Review by h314to — APPROVED** [review]

**Pull request merged**